Update azure-stack-rotate-secrets.md

Paxilein opened the pull request:
The customers can't navigate to shares from ERCS-VM or open a powershell ISE console on the ERCS-VMs. So this is not possible to do without unlocking a supportsession first.

azurecla commented:
Hi __@DileepaKariyawasam__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@DileepaKariyawasam : Thanks for your contribution to the Azure documentation! The author, @rayne-wiselman, has been notified to review your proposed change.

rayne-wiselman commented:
@tysonn 
merge-as-is

Hi @DileepaKariyawasam, thanks for your fix, We'll get this merged.

Rayne

DileepaKariyawasam commented:
Hi Rayne,

No problems at all. Happy to help!

Dileepa

Commit:
Update azure-stack-rotate-secrets.md

The customers can't navigate to shares from ERCS-VM or open a powershell ISE console on the ERCS-VMs. So this is not possible to do without unlocking a supportsession first.

mattbriggs reviewed (approved):
#sign-off
